src/dotnet-new3/ParserFactory.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#nullable enable
+
+using System.CommandLine;
+using System.CommandLine.Builder;
+using System.CommandLine.Help;
+using System.CommandLine.Parsing;
+using Microsoft.TemplateEngine.Cli.Commands;
+
+namespace Dotnet_new3
+{
+    internal static class ParserFactory
+    {
+        internal static Parser CreateParser(Command command, bool disableHelp = false)
+        {
+            var builder = new CommandLineBuilder(command)
+            //.UseExceptionHandler(ExceptionHandler)
+            //.UseLocalizationResources(new CommandLineValidationMessages())
+            //TODO: decide if it's needed to implement it; and implement if needed
```

Comment:
https://github.com/dotnet/templating/issues/2191: implement [parse] and [suggest] directives